common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -1051,7 +1051,7 @@ def allowed_metadata_by_category(category):
     return {
         'vertical': [],
         'chapter': ['start'],
-        'sequential': ['due', 'format', 'start', 'graded']
+        'sequential': ['due', 'due_num_weeks', 'format', 'start', 'graded']
```

Comment:
I don't think this should hurt, but the whole xml import/export process is going to be its own beast I fear.